### PR TITLE
Merge Generic DB framework  changes for OpenLdap to Kubernetes branch

### PR DESCRIPTION
**Pull request by udayan-y21**
# Problem Statement

F45J related changes for Generic DB framework to support Ldap related DB operations

# Design
-  For Bug describe the fix here. 
-  For Feature, Post the link to the solution page on the confluence CORTX Foundation Library 

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: 
-  Confirm All CODACY errors are resolved [Y/N]: 

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: Y
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: Y
- [ ] Confirm Testing was performed with installed RPM [Y/N]:  Y

# Review Checklist 
  Before posting the PR please ensure
- [ ] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [ ] Jira is updated
- [ ] Check if the description is clear and explained. 
- [ ] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

File: py-utils/src/utils/data/db/examples/openldap/cortxuser_model.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+# CORTX-Py-Utils: CORTX Python common library.
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```
Comment (sachinpunadikar) [inline py-utils/src/utils/data/db/examples/openldap/cortxuser_model.py:4]: correct the copyright year

Comment (udayan-y21) [reply py-utils/src/utils/data/db/examples/openldap/cortxuser_model.py:4]: Modifications done.

Comment (sachinpunadikar) [reply py-utils/src/utils/data/db/examples/openldap/cortxuser_model.py:4]: 2020 -> 2021

File: py-utils/src/utils/data/db/examples/openldap/openldap_storage.py [hunk 2]
```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python3
+
+# CORTX-Py-Utils: CORTX Python common library.
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```
Comment (sachinpunadikar) [inline py-utils/src/utils/data/db/examples/openldap/openldap_storage.py:4]: 2020 -> 2021

File: py-utils/src/utils/data/db/openldap/__init__.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+# CORTX-Py-Utils: CORTX Python common library.
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```
Comment (sachinpunadikar) [inline py-utils/src/utils/data/db/openldap/__init__.py:4]: 2020 -> 2021

File: py-utils/src/utils/data/db/openldap/storage.py [hunk 4]
```diff
@@ -0,0 +1,544 @@
+#!/usr/bin/env python3
+
+# CORTX-Py-Utils: CORTX Python common library.
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```
Comment (sachinpunadikar) [inline py-utils/src/utils/data/db/openldap/storage.py:4]: 2020 ->2021

**Review by sachinpunadikar — CHANGES_REQUESTED** [review]
Please work on comments.
Add copyright text to empty files

**Review by sachinpunadikar — APPROVED** [review]
+1

**Pull request merged**